x-pack/plugins/security_solution/public/common/mock/mock_detail_item.ts

```diff
@@ -46,29 +46,29 @@ export const mockDetailItemData: TimelineEventsDetailsItem[] = [
     values: ['siem-kibana'],
   },
   {
-    field: 'agent.id',
-    originalValue: '5de03d5f-52f3-482e-91d4-853c7de073c3',
-    values: ['5de03d5f-52f3-482e-91d4-853c7de073c3'],
+    field: 'cloud.project.id',
```

Comment:
I "de-alphabetized" this mock data to ensure that our function puts it in alphabetical order